frontend/src/__tests__/cypress/cypress/e2e/modelRegistry/ModelRegistry.cy.ts

```diff
@@ -14,8 +20,31 @@ const initIntercepts = ({ disableModelRegistryFeature = false }: HandlersProps)
     }),
   );
   cy.interceptOdh('GET /api/components', { query: { installed: 'true' } }, mockComponents());
+
+  cy.intercept(
+    '/api/k8s/apis/modelregistry.opendatahub.io/v1alpha1/modelregistries',
+    mockK8sResourceList([mockModelRegistry({})]),
+  );
+
+  cy.interceptK8s(ModelRegistryModel, mockModelRegistry({}));
+
+  cy.interceptK8s(
+    RouteModel,
+    mockRouteK8sResourceModelRegistry({ name: 'modelregistry-sample', namespace: 'shared' }),
+  );
+
+  cy.interceptOdh(
+    'POST /api/proxy/api/model_registry/v1alpha2/registered_models',
```

Comment:
the test worked when I changed it to `GET /api/model_registry/v1alpha2/registered_models`

Reply:
This is because the proxy got removed but model registry context wasn't updated and a new model registry proxy service wasn't put in place... 

We're going to have to fix this asap.

Reply:
I expect the correct path will end up being `GET /api/service/model_registry/api/model_registry/v1alpha2/registered_models` once the new proxy is in place.